.github/workflows/ci.yml

```
@@ -19,9 +19,23 @@ jobs:
         with:
           version: 8
       - run: pnpm install
-      - run: pnpm lint  # Runs ESLint
+      - run: pnpm lint # Runs ESLint
       - run: pnpm test:prettier # Runs Prettier check
 
+  jest:
+    name: Run Jest Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)